### PR TITLE
Bug 2098261: Remove BlockPools tab in external mode

### PR DESCRIPTION
**Pull request by rexagod**
Remove BlockPools tab when running in external mode.

Signed-off-by: Pranshu Srivastava <rexagod@gmail.com>
***
<details>
<summary>Screenshot</summary>

#### For external mode.
Screenshot from 2022-06-17 23-40-05

#### For internal mode.
image

</details>

### CODE REVIEW

File: packages/ocs/dashboards/odf-system-dashboard.tsx [hunk 1]
```diff
@@ -56,7 +57,10 @@ const ODFSystemDashboard: React.FC<ODFSystemDashboardPageProps> = ({
         },
       ]);
     }
-  }, [isCephAvailable, pages, setPages, t]);
+    if (isBlockPoolAdded && isExternal) {
```
Comment (bipuladh) [inline packages/ocs/dashboards/odf-system-dashboard.tsx:60]: Can you check if `isExternal` is undefined or false and do it in the last block itself. 
Something like `isExternal === undefined ? false : !isExternal`

**Comment by rexagod** [comment]
Flaky: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/red-hat-storage_odf-console/304/pull-ci-red-hat-storage-odf-console-master-odf-console-e2e-aws/1539205155314995200/artifacts/odf-console-e2e-aws/tests/artifacts/gui-test-screenshots/screenshots/add-capacity.spec.ts/. This code does not affect the add-capacity flow.

**Pull request merged**